Added status texts

dima-bzz opened the pull request:
Signed-off-by: Dmitry Mazurov <dimabzz@gmail.com>

Ionaru commented:
Hi @dima-bzz, thank you for you contribution to this project!

Can you give me a short summary of the changes you have made in this PR? I'm all aboard with supporting multilanguage status texts, but I'm not sure what the reason is for the other changes you have made. Please explain them so I can understand.

dima-bzz commented:
@Ionaru I added the ability to translate the status line, namely lines, words and Autosaved. Autosaved was added in the previous PR, but here I decided to combine the translation of the status line and therefore removed the text from the autosaved option. I also added an update to the cursor position when I click on the text.

dima-bzz commented:
@Ionaru Can make a separate json file with translations?
